src/NzbDrone.Core/CustomFormats/CustomFormatCalculationService.cs

```diff
@@ -153,21 +156,24 @@ private static List<CustomFormat> ParseCustomFormat(CustomFormatInput input, Lis
             return matches.OrderBy(x => x.Name).ToList();
         }
 
-        private static List<CustomFormat> ParseCustomFormat(EpisodeFile episodeFile, Series series, List<CustomFormat> allCustomFormats)
+        private List<CustomFormat> ParseCustomFormat(EpisodeFile episodeFile, Series series, List<CustomFormat> allCustomFormats)
         {
             var releaseTitle = string.Empty;
 
             if (episodeFile.SceneName.IsNotNullOrWhiteSpace())
             {
                 releaseTitle = episodeFile.SceneName;
+                _logger.Trace("Using scene name for release title: '{0}'", releaseTitle);
```

Comment:
Logging should go above the assignment (same for the others)

Reply:
Updated and code duplicated accordingly due to the reorder.

Reply:
Alt option:
- trace log the source (existing sans release title)
- then log release title as debug or trace at the end once assigned